### PR TITLE
Co-located table | Establish table for co-located users

### PR DESCRIPTION
**Pull request by evankroske**
- Fixes https://github.com/department-of-veterans-affairs/caseflow/issues/6077

## Description
Adds support for colocated admins to /queue. Today, the page displays all of their tasks, independent of whether they are on hold. Also, the table is missing the column "Type(s)" because the information required by that column isn't yet available through the tasks API. [Github issue](https://github.com/department-of-veterans-affairs/caseflow/issues/6468).

### Screenshots
screen shot 2018-08-02 at 11 43 06 am

## Testing Plan
- [x] Become BVALSPORER.
- [x] Go to /queue.
- [x] Confirm that task information is displayed.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -170,42 +170,42 @@ def create_ama_appeals
     @ama_appeals << FactoryBot.create(
       :appeal,
       advanced_on_docket: true,
-      veteran_file_number: "209179363",
-      veteran: FactoryBot.create(:veteran),
+      veteran_file_number: "701305078",
+      veteran: Veteran.find_by(file_number: "701305078"),
```
Comment (mdbenjam) [inline db/seeds.rb]: I don't think you need to specify the veteran, if you're just going to find an existing record.

File: client/app/queue/ColocatedTaskListView.jsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      <AmaTaskTable tasks={this.props.amaTasks} />
+    </div>;
+
+    return <AppSegment filledBackground>
+      {tableContent}
+    </AppSegment>;
+  };
+}
+
+ColocatedTaskListView.propTypes = {
```
Comment (mdbenjam) [inline client/app/queue/ColocatedTaskListView.jsx]: Can probably remove this

File: client/app/queue/ColocatedTaskListView.jsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  componentDidMount = () => {
+    this.props.clearCaseSelectSearch();
+    this.props.resetErrorMessages();
+  };
+
+  render = () => {
+    const { error, success } = this.props;
+    const tableContent = <div>
+      <h1 {...fullWidth}></h1>
+      {error && <Alert type="error" title={error.title} scrollOnAlert={false}>
```
Comment (mdbenjam) [inline client/app/queue/ColocatedTaskListView.jsx]: Do we need all this `Alert` stuff? Presumably we will once we hook up the flows for colocated to assign things back to judges?

File: db/seeds.rb (hunk 1 above)
Comment (evankroske) [reply db/seeds.rb]: Thanks for pointing this out! Fixed!

File: client/app/queue/ColocatedTaskListView.jsx (hunk 3 above)
Comment (evankroske) [reply client/app/queue/ColocatedTaskListView.jsx]: Nope. I removed it and a bunch of other leftover code in ColocatedTaskListView

File: client/app/queue/ColocatedTaskListView.jsx (hunk 2 above)
Comment (evankroske) [reply client/app/queue/ColocatedTaskListView.jsx]: Done!

**Pull request merged**